### ISSUE TITLE
Light competitive analysis of federal and other design systems' research sections

### ISSUE THREAD

**Issue by finekatie** [uswds/uswds-site#2340]
Light competitive analysis of federal and other design systems' research sections to help determine best practices for USWDS Research page

**Comment by finekatie** [comment]
Created with Sarah Schroeder's help today :)

**Comment by finekatie** [comment]
Started this on Thursday.

**Comment by finekatie** [comment]
Likely move this into next sprint.

**Comment by finekatie** [comment]
This has been sitting, waiting for me to do some work on it :(

**Comment by finekatie** [comment]
Had already taken some notes. Neither UK, Scottish or Canadian design systems have independent "Research" pages. All research activities are contained to the components or patterns, with links to GitHub when applicable. Canada doesn't have research mentioned at all, and actually may be an outdated design system.

**Comment by finekatie** [comment]
Sarah and I discussed the quick looks that I took and my comments above should suffice for this being complete unless there is more desired.
